### PR TITLE
Add proudly maintainers of page on the homepage w/ Yash Vardhan <yash…

### LINKED ISSUE #1421: Implementation of Proudly maintainers and contributors section on homepage
**Current Behavior**
We currently have no implementation of the  Proudly maintainers and contributors section on homepage 

**Desired Situation**
The design of the home page is currently approved on figma, and we will appreciate someone jumping on the task of implementing it 

https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=1869%3A2967

Check out the design below
proudly

[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Grumpyyash**
…vardhan513@gmail.com>

Signed-off-by: Yash Vardhan <>

Added the "proudly maintainers of" section on the homepage.

This PR fixes #1421 
final

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
@leecalcote which all service meshes should we list here?

File: src/sections/Proud-maintainers/index.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
+import React from "react";
+import { Link } from "gatsby";
+
+import { Container, Row, Col } from "../../reusecore/Layout";
+import SectionTitle from "../../reusecore/SectionTitle";
+import Button from "../../reusecore/Button";
+
+import Image1 from "../../assets/images/app/proudMaintainers/envoy.png";
```
Comment (Nikhil-Ladha) [inline src/sections/Proud-maintainers/index.js:8]: Don't create the directory inside `app`, keep it under `images/proudMaintainers` only.

Comment (Grumpyyash) [reply src/sections/Proud-maintainers/index.js]: Updated!

**Comment by Nikhil-Ladha** [comment]
The responsiveness is an issue, but waiting for an update from Lee...then it can be updated.

File: src/sections/Proud-maintainers/proudMaintainers.style.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+            }
+        }
+
+     }
+     @media only screen and (max-width: 480px) {
+        .project__block__inner{
+            padding: 30px;
+        }
+     }
+
```
Comment (Nikhil-Ladha) [inline src/sections/Proud-maintainers/proudMaintainers.style.js:166]: Remove extra lines.

File: src/sections/Proud-maintainers/index.js [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+                <Col sm={6} md={6} lg={6}>
+                  <img src={Image4} alt="" />
+                </Col>
+                <Col className="company-name" sm={6} md={6} lg={6}>
+                  <h4>Traefik</h4>
+                </Col>
+              </Row>
+            </div>
+          </Col>
+
```
Comment (Nikhil-Ladha) [inline src/sections/Proud-maintainers/index.js:107]: Remove these extra lines as well.

Comment (Nikhil-Ladha) [reply src/sections/Proud-maintainers/index.js:107]: Also, please check and remove them from other places as well.

File: src/sections/Proud-maintainers/index.js (hunk 1 above)
Comment (leecalcote) [reply src/sections/Proud-maintainers/index.js]: Fixed

File: src/sections/Proud-maintainers/proudMaintainers.style.js (hunk 2 above)
Comment (leecalcote) [reply src/sections/Proud-maintainers/proudMaintainers.style.js]: Fixed

**Comment by leecalcote** [comment]
Some work needs done on vertical alignment, margins, padding and responsiveness.

**Comment by Nikhil-Ladha** [comment]
@Grumpyyash the update is done....can you fix the responsiveness now....also please check the issue with the path which is causing the build failure.

**Comment by leecalcote** [comment]
Merging this PR, because its changes are needed elsewhere. Opened an issue to fix vertical alignment, margins, padding and responsiveness issues.

**Pull request merged**